Toggl.iOS/Presentation/NavigationPresenter.cs

```diff
@@ -11,6 +11,7 @@ public sealed class NavigationPresenter : IosPresenter
     {
         protected override HashSet<Type> AcceptedViewModels { get; } = new HashSet<Type>
         {
+            typeof(AboutViewModel),
             typeof(CalendarSettingsViewModel),
             typeof(ForgotPasswordViewModel),
             typeof(NotificationSettingsViewModel),
```

Comment:
We need to add this too 🙈 

```suggestion
            typeof(LicensesViewModel),
            typeof(NotificationSettingsViewModel),
```